src/test/java/se/sundsvall/supportmanagement/api/model/errand/ErrandTest.java

```diff
@@ -63,6 +64,7 @@ void testBuilderMethods() {
 		final var contactReason = "reason";
 		final var contactReasonDescription = "contactReasonDescription";
 		final var labels = List.of("label");
+		final var notifications = List.of(Notification.create().withErrandId("errandId").withType("type").withDescription("description").withCreatedBy("createdBy").withCreated(OffsetDateTime.now()));
```

Comment:
Do you need to set all these properties? 
They are not asserted (and doesn't need to be asserted here).

I suppose that thiis is sufficient:
`List.of(Notification.create())`